data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -138,11 +138,11 @@ describe('POST', () => {
             .expect('Content-Type', /json/)
             .expect(201);
     });
-    it('create with invalid input and validate_only should return 422', async () => {
+    it('create with input missing required properties and validate_only should return 400', async () => {
         return request(app)
             .post('/api/cases?validate_only=true')
             .send({})
-            .expect(422);
+            .expect(400);
     });
```

Comment:
Is there a test that _has_ required parameters, but there's an invalid value, that returns 422?

Reply:
added one -- the distinction is that 400's come from the API validation, and 422's come from Mongoose validation

so added a test that (at least for now) we're only checking at the mongoose layer (age range)

Reply:
Cool, SG. We should reflect on which error is raised when afterwards, but this is great for now.